src/sagas/networkSettings.js

```diff
@@ -202,12 +202,11 @@ export function* updateNetworkSettings(action) {
   // Validates the potential network and set the network accordingly
   if (potentialNetwork === NETWORK_MAINNET) {
     network = NETWORK_MAINNET;
-  } else if (potentialNetwork.startsWith(NETWORK_TESTNET)) {
+  } else if (potentialNetwork.indexOf(NETWORK_TESTNET) >= 0) {
     network = NETWORK_TESTNET;
-  } else if (potentialNetwork.startsWith(NETWORK_PRIVATENET)) {
+  } else if (potentialNetwork.indexOf(NETWORK_PRIVATENET) >= 0) {
     network = NETWORK_PRIVATENET;
   } else {
-    log.debug('The network informed is not allowed. Make sure your network is either "mainnet", "testnet" or "privatenet", or starts with "testnet" or "privatenet".');
     yield put(networkSettingsUpdateFailure());
     return;
```

Comment:
I've been using the following code in the desktop (and in the lib method that will substitute this code here)

```
if (potentialNetwork === NETWORK_MAINNET) {
  network = NETWORK_MAINNET;
} else if (potentialNetwork.includes(NETWORK_TESTNET)) {
  network = NETWORK_TESTNET;
} else {
  network = NETWORK_PRIVATENET;
}
```

I think we can continue with your code but there's a small difference and the code I wrote above will substitute this one soon with the lib method.